### PR TITLE
add release-tools into master.

### PR DESCRIPTION
**Pull request by MeinhardZhou**
1. add release-tools into master.
2. add .prow.sh.
3. add a symlink in Makefile to related to release-tools.
4. modify some code style for format test.

### CODE REVIEW

File: release-tools/OWNERS [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 approvers:
 - kubernetes-csi-approvers
-- pohly
+- meinhardzhou
```
Comment (pohly) [inline release-tools/OWNERS:5]: You cannot modify release-tools in the [csi-driver-nvmf](https://github.com/kubernetes-csi/csi-driver-nvmf) repo. You should use it verbatim and only modify files outside of it.

**Review by pohly — CHANGES_REQUESTED** [review]

File: release-tools/OWNERS (hunk 1 above)
Comment (MeinhardZhou) [reply release-tools/OWNERS]: Done!

**Review by pohly — COMMENTED** [review]
This looks good to me.

It might make sense to to create the corresponding Prow job(s) first, then we can test this PR with them.

Testing of this driver in a kind cluster probably doesn't work, does it? That leaves the unit testing and image building jobs. For that you can add "csi-driver-nvmf" to https://github.com/kubernetes/test-infra/blob/c5c41b9c85b0a342c2ed1c2622e3820536e09ad2/config/jobs/kubernetes-csi/gen-jobs.sh#L93

Then run the script and submit a PR against test-infra.

**Comment by MeinhardZhou** [comment]
> 

https://github.com/kubernetes/test-infra/pull/25445

I have already created this PR. :)

**Comment by MeinhardZhou** [comment]
> /test pull-kubernetes-csi-csi-driver-nvmf

It seems to be because this repo does not have prow.sh. Do I add prow.sh support to this pr, or submit on another pr?

**Review by pohly — CHANGES_REQUESTED** [review]
`.prow.sh` must be a symlink to `release-tools/prow.sh`.

**Comment by pohly** [comment]
Let's add that in this PR.

**Comment by pohly** [comment]
Looks like more thorough testing now finds several problems. Let's fix those in this PR.

**Comment by pohly** [comment]
Yes, please clean up the commits. The code changes probably should come first, with a commit message that explains why they are done.

Then follow with one commit that has the local changes for integrating csi-release-tools.

**Comment by pohly** [comment]
I can't comment on the code changes, but the rest looks good to me.

**Comment by MeinhardZhou** [comment]
ok, do I need your lgtm or I can type this command myself?

**Comment by pohly** [comment]
You need some other reviewer for the code who then adds the LGTM.

I understand that this can be tricky when only a single person knows the code or has the time to look at it. Perhaps raise this on Slack and ask for help?

**Comment by xing-yang** [comment]
@MeinhardZhou any reason why the additional code changes have to be in the same PR as the release-tools changes?  At least the PR title and PR description need to be updated to explain this.

**Comment by MeinhardZhou** [comment]
> @MeinhardZhou any reason why the additional code changes have to be in the same PR as the release-tools changes? At least the PR title and PR description need to be updated to explain this.

@xing-yang when I updated code for the anti-test job, I found some code naming and code structure is not very reasonable, so I want to modify them all in this pr, and it won't change the main logic of the driver.

**Comment by xing-yang** [comment]
@MeinhardZhou It's better to submit the code changes in a separate PR.

**Comment by MeinhardZhou** [comment]
> @MeinhardZhou It's better to submit the code changes in a separate PR.

I'm done :)

File: .prow.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#! /bin/bash
+
+# Copyright 2020 The Kubernetes Authors.
```
Comment (xing-yang) [inline .prow.sh:3]: 2022

**Pull request merged**